numpy/core/tests/test_numeric.py

```diff
@@ -1530,7 +1530,7 @@ def test_simple_inplace_02(self):
         m = -0.5
         M = 0.6
         self.fastclip(a, m, M, a)
-        self.clip(a, m, M, ac)
+        self.clip(ac, m, M, ac)
```

Comment:
Was there a bug in the test here before then?

Reply:
Yeah, fastclip modifies `a` in place, this `self.clip` is actually a manual clip in the test. Looks wrong to me, can put it into its own commit.